apps/examples/testcase/le_tc/network/tc_net_poll.c

```diff
@@ -0,0 +1,183 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/*  @file tc_net_poll.c
+ * @brief Test Case Example for lwip_poll() API
+ */
+
+#include <stdio.h>
+#include <netinet/in.h>
+#include <netutils/netlib.h>
+#include <sys/socket.h>
+#include <lwip/sockets.h>
+#include <tinyara/os_api_test_drv.h>
+#include <sys/ioctl.h>
+#include <poll.h>
+#include "tc_internal.h"
```

Comment:
Please follow below header format:

##HEADEER FORMAT
<tinyara/config.>
standard headers
TizenRT(TinyAra) specific headers
framework or external headers
Local headers
Please place the header file like below sequence.
************************
tinyara/config.h
standard header : stdio.h, unistd.h, string.h and so on.
tinyara/xxx.h
absolute path header : "xxx.h"
************************

Reply:
corrected the order and we did n't need of tinyara/config.h file.
so didn't include.